test/UAP.test.ts

```diff
@@ -103,7 +101,7 @@ describe("UniversalReceiverDelegateUAP", function () {
   });
   describe("universalReceiverDelegate", function () {
 
-  it("should proceed with super function if no type configuration is found", async function () {
+  it.only("should proceed with super function if no type configuration is found", async function () {
```

Comment:
@democrazyfi we should run all the tests

Reply:
opps done